<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information</class>
 <widget class="QWidget" name="Information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>个人信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="username_edit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="name_edit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="sex_edit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="id_number_edit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="phone_edit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>身份证号</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QLineEdit" name="email_edit">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="update_button">
        <property name="text">
         <string>修改信息</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/修改.png</normaloff>:/new/icon/icon/修改.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="sure_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/确定.png</normaloff>:/new/icon/icon/确定.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="update_password_button">
        <property name="text">
         <string>修改密码</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/密码.png</normaloff>:/new/icon/icon/密码.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>邮箱</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="cancel_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/取消.png</normaloff>:/new/icon/icon/取消.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>乘车人</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="passenger_table">
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <column>
         <property name="text">
          <string>姓名</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>性别</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>身份证号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>手机号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>类型</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="change_passenger_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>修改</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/修改.png</normaloff>:/new/icon/icon/修改.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_passenger_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/删除.png</normaloff>:/new/icon/icon/删除.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_passenger_button">
        <property name="text">
         <string>添加</string>
        </property>
        <property name="icon">
         <iconset resource="qss.qrc">
          <normaloff>:/new/icon/icon/添加.png</normaloff>:/new/icon/icon/添加.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qss.qrc"/>
 </resources>
 <connections/>
</ui>
